docs/gettinginvolved/plugins/faq.rst

```diff
@@ -33,15 +33,14 @@ Is there a script to cleanup volumes leftover from running functional tests?
 -----------------------------------------------------------------------------
 
 Yes.
-After each test case, `detach_destroy_volumes <https://github.com/ClusterHQ/flocker/blob/master/flocker/node/agents/test/test_blockdevice.py>`_ is run automatically to cleanup volumes created by the test case.
-This cleanup call is added as part of `get_blockdeviceapi_with_cleanup <https://github.com/ClusterHQ/flocker/blob/master/flocker/node/agents/test/blockdevicefactory.py>`_ .
+After each test case, ``flocker.node.agents.testtools.detach_destroy_volumes`` will cleanup volumes created by the test case.
```

Comment:
Hrm. I think this sentence maybe should read "After each test case, .... can be registered as a cleanup function to cleanup volumes by the test case." Just because it will only cleanup volumes if wrapper is used.